### ISSUE TITLE
[Topic] Design checks for web accessibility

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#172]
URL: https://wpaccessibility.org/docs/testing/design/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/testing/design/index.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

This content should be reviewed and restructured.
The links need to be checked and updated.
